docs/pages/admin-guides/deploy-a-cluster/helm-deployments/ibm.mdx

```diff
@@ -0,0 +1,324 @@
+
+Wait until the provisioning is over and the database ready. This can take several minutes.
+
+On the postgres resource page:
+- download the database TLS certificate, this will be used by Teleport to connect to postgres.
+- copy the database hostname and port, we will need them to configure Teleport
+
+![PostgreSQL endpoints screenshot](../../../../img/ibm/get-postgres-connstring.png)
+
+## Step 2/7. Create the storage bucket
+
+In this step, we'll create the object storage backend Teleport will use to store its session recordings.
+
+Select *Create Resource*, search *object*, and select *Object Storage*.
+Fill the storage creation form with the following settings:
+
+- IBM Cloud-hosted storage
+- Standard pricing plan
+
+Then, chose *Create bucket*, *Create a Custom Bucket*, and fill the bucket creation form with the following settings:
+- Regional Resiliency
+- Smart Tier
+- Disabled Object Versioning
+- (Optional) configure your retention policy. This configures how long session recordings are kept. Leave empty for infinite retention.
+
+Open the bucket *Configuration* and save the direct endpoint, this will be used to configure Teleport to use the bucket.
+
+Navigate the object storage instance and create a new service credential with HMAC:
+![Cloud object storage credential creation screenshot](../../../../img/ibm/create-bucket-credentials.png)
+
+Open the credentials and save the content of the `access_key_id` and `secret_access_key` fields.
+This will be needed later to authorize Teleport to read and write session recordings in the bucket.
+
+## Step 3/7. Configure the postgres Database
+
+In this step we will configure two users on the postgres database:
+- a `teleport` user that will create and own the `teleport_backend` and `teleport_audit` databases
+- a `repl` user, which is the only user allowed to read the stream of database changes. Teleport uses it to listen for
+  database changes and synchronize the multiple Teleport Auth Service replicas.
+
+Open a terminal and:
+- install the `ibmcloud` database plugin
+  ```code
+  $ ibmcloud install plugin cdb
+  ```
+- recover the postgresql database deployment name
+  ```code
+  $ ibmcloud cdn deployments
+
+  Name               Location   State
+  teleport-backend   ca-tor     active
+  ```
+- configure the WAL settings of the database
+  ```code
+  $ ibmcloud cdb deployment-configuration teleport-backend '{"wal_level":"logical", "max_replication_slots":30, "max_wal_senders":30}'
+  ```
+- configure a password for the replication user
+  ```code
+  # generate a random password
+  $ export POSTGRES_PASSWORD="$(tr -dc A-Za-z0-9_ < /dev/urandom | head -c 32 )"
+
+  # display the password
+  echo "$POSTGRES_PASSWORD"
+
+  # set the password
+  $ ibmcloud cdb deployment-user-password teleport-backend repl "$POSTGRES_PASSWORD"
+  ```
+- create a `teleport` user and set its password
+  ```code
+  # set the same password as the replication user
```

Comment:
Are we ok with this? We could have a passfile with different passwords for the two users, which would also get rid of the global PGPASSWORD envvar.